### PR TITLE
Add API token authorization details

### PR DESCRIPTION
**Pull request by dacharyc**
Tagging you for review, Ben, because this documentation is for your "needs documentation" ticket (I'm about to archive it) and tagging you, James, because this info is mostly from your excellent gist and I'd love you to review for accuracy/details.

### CODE REVIEW

**Comment by bchavet** [comment]
> I was expecting to see this new information in this section, since it's directly related. If you're going through the process outlined here and don't have those options, then you need to follow these instructions instead.

I think the actual instructions make sense where @dacharyc put them, but it would be good to include a summary and link to them from the troubleshooting section.

File: content/setting-up-tugboat/connect-with-your-provider.md [hunk 1]
```diff
@@ -274,3 +275,103 @@ You'll see a dialog box explaining what happens if you disconnect a linked accou
 ![Dialog box explaining disconnecting a linked git provider account with arrow to "Disconnect" button](../../_images/press-disconnect-button.png)
 
 {{% /expand%}}
+
+## Authenticating using an API access token
+
+By default, Tugboat uses OAuth to authenticate to git hosting providers. However, there may be times when GitHub,
+GitLab, or Bitbucket organizations take too long to approve org-level OAuth applications. As a workaround, you can use a
```
Comment (bchavet) [inline content/setting-up-tugboat/connect-with-your-provider.md:282]: This makes it sound like GH, GL, or BB can take a long time to authenticate OAuth applications. That's not really the case. The problem we are working around is when an organization admin can't or won't approve Tugboat at the org-level. Without that approval, the repo selector in the web interface will not work.

File: content/setting-up-tugboat/connect-with-your-provider.md [hunk 2]
```diff
@@ -274,3 +275,103 @@ You'll see a dialog box explaining what happens if you disconnect a linked accou
...
+git provider's API access token to add a GitHub, GitLab, or Bitbucket repo to Tugboat.
+
+You'll need to use the [Tugboat API](https://api.tugboat.qa/) to pass an alternate authentication method.
+
+This example shows how to implement this workaround for a GitHub repository, but you can do the same thing using a
+slightly different syntax for GitLab or Bitbucket repositories. If you need help with the syntax for GitLab or
+Bitbucket, or have questions about this workaround, reach out to us for [support](/support).
```
Comment (bchavet) [inline content/setting-up-tugboat/connect-with-your-provider.md:289]: Can we link to the API documentation here where we describe this endpoint, in addition to the support link?

https://api.tugboat.qa/v3#tag/Repositories/paths/~1repos/post

**Comment by bchavet** [comment]
> Am I missing something that would warrant including these instructions as "troubleshooting" steps versus an alternate authorization option?

This would likely present itself as a "Missing Repository", because we can only list repos for orgs that we are authorized for as an OAuth app.

I still think it belongs where you put it. But I could see how a quick blurb in the Troubleshooting section with a link back to the full documentation for it would be helpful.

**Review by bchavet — DISMISSED** [review]
Looks good to me!

**Comment by dacharyc** [comment]
> > Am I missing something that would warrant including these instructions as "troubleshooting" steps versus an alternate authorization option?
> 
> This would likely present itself as a "Missing Repository", because we can only list repos for orgs that we are authorized for as an OAuth app.
> 
> I still think it belongs where you put it. But I could see how a quick blurb in the Troubleshooting section with a link back to the full documentation for it would be helpful.

Alrighty, good to know! I've reworked the existing explanation a little, and added a blurb with a link to the new section. Does this accurately describe the potential states & how/why they occur, @bchavet ?

**Review by bchavet — APPROVED** [review]

**Pull request merged**